README.md

```diff
@@ -42,10 +42,10 @@ fn users_with_name(connection: &Connection, target_name: &str)
     -> Vec<(i32, String, Option<String>)>
 {
     use self::users::dsl::*;
-    users.filter(name.eq(target_name))
-        .load(connection)
-        .unwrap()
-        .collect()
+    match users.filter(name.eq(target_name)).load(connection) {
+        Ok(results) => results.collect(),
+        Err(_) => vec![]
+    }
```

Comment:
`QueryResult<Vec<(...)>>` is awkward here because `load` returns `QueryResult<Cursor<_,_>>`.

Would the following be okay instead?

```rust
fn users_with_name(connection: &Connection, target_name: &str)
    -> Option<Vec<(i32, String, Option<String>)>>
{
    use self::users::dsl::*;
    users.filter(name.eq(target_name)).load(connection)
        .ok()
        .map(|x| x.collect())
}
```